src/containers/TableWrapper.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Link } from 'react-router-dom'
+import { farmRef, locationRef } from 'API/databases.js'
+import GeoFire from 'geofire'
+import setGeoQuery from 'actions/Geoquery'
+import GetFarms from 'components/GetFarms'
+
+const mapStateToProps = state => {
+  return {
+    failure: state.geocodeReducer.failure
+  }
+}
+
+const mapDispatchToProps = dispatch => {
+  return {
+    updateQueryInfo: (type, address, userCoords, range) =>
+      dispatch(setGeoQuery(type, address, userCoords, range))
+  }
+}
+
+class TableWrapper extends Component {
+  constructor(props) {
+    super(props)
+    this.geoQuery = ''
+    this.location = this.userCoords()
+    this.state = {
+      keys: [],
+      rawFarmData: [],
+      error: ''
+    }
+  }
+
+  // TODO: clean up function
+  // Takes location from URL and returns object containing location information
+  userCoords = () => {
+    const S = this.props.match.params.location
+    const lat = Number(S.substring(S.indexOf('LAT=') + 4, S.indexOf('&LNG')))
```

Comment:
We can parse these better using a library that parses URL parameters. That way the parameters can be passed in any order. Check out something like [query string](https://www.npmjs.com/package/query-string)